storagedriver/factory/factory.go

```diff
@@ -58,7 +58,12 @@ func Create(name string, parameters map[string]interface{}) (storagedriver.Stora
 		// }
 		// return driverClient, nil
 	}
-	return driverFactory.Create(parameters)
+	d, err := driverFactory.Create(parameters)
```

Comment:
My mistake. I meant this location is fine but it would be better that they are in the `New` functions of each driver so their is no possibility to create a without the wrapper. Many of the unit tests use the `New` function directly and we should be able to continue to use that to can get a valid driver.